### DISCUSSION TITLE
Vimar Zigbee Devices - Not responding properly

### DISCUSSION THREAD

**Discussion by huymalo** [Koenkk/zigbee2mqtt#17472] [Devices]
Hello,

Quite new to zigbee2mqtt. I have a few Vimar branded units that I'd like to implement but they don't seem to work correctly. I believe they are supported, models:
[19592.0](https://www.vimar.com/en/int/catalog/product/index/code/19592.0)
[19594.0](https://www.vimar.com/en/int/catalog/product/index/code/19594.0)

The issue i'm having is the following:
1. Difficulty adding devices. Took many attempts, is this normal?
2. No feedback. When I toggle the relay, I do not get feedback in the debug log
3. In the frontend, when I set the state to On, the relay triggers then the frontend reverts to Off. (No feedback)

Any advice? Thanks for the help

**Comment by huymalo** [comment]
Anyone?

**Reply by federicotravaini** [reply]
I am facing the same problems and can't figure out how to solve them. I am using the VIMAR LINEA units but it's so unreliable that I am switching them of until I find a proper solution. They switch on during the day randomly, I can't control them, they have wrong state. Literally a nightmare.

**Reply by Sim0cYz** [reply]
Hi Federico, I have several Vimar Linea devices (roller shutter actuator, light switch, dimmer) and they work great with zigbee2mqtt and Sonoff ZBDongle-E. Sad thing is that they make all the other zigbee devices act weird (unresponsive, slow). Also I have address conflict errors since I added them to my existing zigbee network (I was previously using them in Bluetooth mode, with their Vimar hub). Specifically I have 4 Sonoff ZBMini that are being constantly kicked out of the network because of address conflicts. They were working flawlessly before migrating all the Vimar stuff on zigbee...

**Reply by federicotravaini** [reply]
Same issue. My zigbee network works just fine but if I had a Vimar Linea device it just stops working reliably after a while. Everything becomes super slow. 

I am using ZHA and SLZB-06.

I will try to create a new zigbee network with only Vimar devices and see if they work together at least.

**Reply by Sim0cYz** [reply]
To be honest, I have this issue only with those 4 Sonoff ZBMini. All the other ~10 devices on the network are working just fine, together with around 10 more devices from Vimar..!

**Reply by Sim0cYz** [reply]
Forgot to specifty, I'm on latest zigbee2mqtt. Today I removed all the Vimar devices from my network, and everything is running fine. No address conflict, no Sonoff being kicked. As soon as I add a Vimar device, for example a light switch, network begins acting weird again.
I paired them again with their bluetooth Vimar Gateway because it's breaking almost everything that relies on zigbee at my house.

**Reply by federicotravaini** [reply]
I tried both Z2M and ZHA and both cases was a disaster.
Now my network has been stable for a week or more when I removed Vimar devices (light).
I ordered the gateway and will get away with it but it's a shame.

**Reply by Sim0cYz** [reply]
> I tried both Z2M and ZHA and both cases was a disaster. Now my network has been stable for a week or more when I removed Vimar devices (light). I ordered the gateway and will get away with it but it's a shame.

I wasted the whole afternoon trying to troubleshoot these devices. Vimar lights, or roller shutter, alone or together, always mess up the zigbee network. I had to reinstall the bluetooth firmware, do a factory reset (press their buttons for 30 seconds after a power cycle) and add them again to the gateway's bluetooth mesh. Then I had to set the timing of each roller shutter. So frustrating, enormous waste of time. Guess they adopted zigbee their own way, and it's a total failure.

Note that this way, at least with HomeKit integration of the Vimar gateway on Home Assistant, "stop" button is not there for roller shutters. There's only up, down and percentage setting.

**Reply by federicotravaini** [reply]
Luckily I only have light buttons.
I got in touch with their assistance but their reply was: "we tested it with Alexa and SmartThings and it works".
I have BTIcino living now light and they work seamlessly with ZHA (tbh with Z2M you get more entities).

When I have time to spare I will try to create a new zigbee network only for vimar devices and see if it works.

**Reply by Sim0cYz** [reply]
No need to waste your time, I tried today. If you create a network only for Vimar devices, they work just fine. Lights are super responsive, and so are the roller shutters. As soon as you add other vendors devices, these will start acting weird. (Vimar devices continue to work pretty good): some entities become unresponsive, some get conflicting addresses and get kicked from the network. It's sad, because my zigbee mesh would be super strong (I have roughly 25 zigbee devices and 12 of them are Vimar, as soon as I remove them my network halves) but it performs so better without them.
I have also crappy 5$ devices from China and none of them act that bad. Guess Vimar implemented Zigbee their own way, and something is wrong.

**Reply by Sim0cYz** [reply]
> I got in touch with their assistance but their reply was: "we tested it with Alexa and SmartThings and it works".

Mind sharing how did you got in touch with them? I'd try to report them these issues one more time.

**Reply by federicotravaini** [reply]
Here: https://www.vimar.com/it/it/contact/form/request?type=technic-support
